Dockerfile

```diff
@@ -9,7 +9,11 @@ ENV DEBIAN_FRONTEND noninteractive \
     SERVER_URL https://localhost:4443 \
     RUNDECK_STORAGE_PROVIDER file \
     RUNDECK_PROJECT_STORAGE_TYPE file \
-    NO_LOCAL_MYSQL false
+    NO_LOCAL_MYSQL false \
+    LOGIN_MODULE RDpropertyfilelogin \
```

Comment:
This is to facilitate ldap auth by just adding 2 params and volume mounting the ldap config file